src/makeCalculator.js

```diff
@@ -5,6 +5,33 @@
  */
 function makeCalculator() {
   // write code here
+  const calculator = {
+    result: 0,
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
+    operate(callback, number) {
+      callback.call(this, number);
```

Comment:
Instead of using `call`, you can directly invoke the callback with `this.result` as the context, as `this` is already bound to the `calculator` object in the methods.